spec/services/post_scraper_spec.rb

```diff
@@ -305,7 +305,7 @@
 
     html = "<!DOCTYPE html>\n<html></html>\n"
     stub_with_body = double
-    expect(stub_with_body).to receive(:body).and_return(html)
+    allow(stub_with_body).to receive(:body).and_return(html)
```

Comment:
What's the difference between allow and expect here? I thought expect would require a call, but allow would just stub it out (without any expectations).

Reply:
Basically that, allow stubs and expect does an expectation. (Expect also stubs In An Order which came up in one of the tests xD) Rubocop thinks you shouldn't mix expecting something and stubbing it and honestly I think it's probably right. ...possibly some of these should *also* be expected :thinking:

Reply:
How do we expect without a stub in context?

Reply:
I think expect to receive doesn't stub but I have to check. that or you do the stubbing desperately which is, at least, clearer as to what the heck you're doing?